### ISSUE TITLE
What's the most common accessibility mistake you see?

### ISSUE THREAD

**Issue by sindresorhus** [jdan/ama#9]

**Comment by jdan** [comment]
The most common mistake is probably not giving your text sufficient contrast. (If your background is #ffffff, small text must be darker than #76767 to meet the WCAG 2.0 guidelines!)

I'm currently typing this answer on my gaming monitor, and even GitHub is hard to read! This one isn't a huge deal-breaker, though, since a lot of folks will be browsing in high-contrast mode anyway.

Similarly, `outline: none` is *all* over the web and bah it's the worst! You need a focus indicator, otherwise your users will lose their place. If you insist on `outline: none`ing, you'll need a good substitute (e.g. `border-bottom`)

If we're talking *breaking* the experience for some users, a big violator is attaching click events to `<div>`s, `<span>`s, and `<a>`s without an href attribute. These are mouse-dependent, and getting them to respond to a keyboard is pretty troublesome (though you *can* do it with the right JS and `aria-*` attributes).
